ios/ios/SplashView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SplashView.swift
+//  ios
+//
+//  Created by Thiên Nguyễn on 8/10/23.
+//  Copyright © 2023 orgName. All rights reserved.
+//
+
+import SwiftUI
+
+let SHOW_LOGO_DELAY = 0.5
+let SHOW_LOGO_DURATION = 1.0
+
+
+struct SplashView: View {
+
+    @State private var isLoaded = false
+
+    var body: some View {
+        ZStack {
+            Image("BackgroundSplash").resizable()
```

Comment:
Please bring `.resizable()` to the new line.

```suggestion
            Image("BackgroundSplash")
            	.resizable()
```